js/NavMainComponent.js

```diff
@@ -74,8 +112,33 @@ NavMainComponent.prototype.switchSecondaryNav = function(target) {
     component.closeSubNavs();
 
     component.$html.find('.nav-list.is-active').removeClass('is-active');
-    component.$html.find('[data-nav="' + target + '"]')
-        .addClass('is-active');
+    component.$html.find('[data-nav="' + target + '"]').addClass('is-active');
+};
+
+NavMainComponent.prototype.switchTertiartyNav = function(target) {
+
+    var component = this;
+
+    component.$html.find('.navTertiary .nav-link').removeClass('is-active');
+
+    if (component.$html.find('[data-nav="' + target + '"]').length < 1) {
+        component.closeNavs();
+    }
+
+    component.$html.find('.navTertiary .is-active').removeClass('is-active');
```

Comment:
A small improvement (may not be worth it) but we've got references up on line ~20 to the `.navTertiary` items, could we just filter that list here? Or do we need to query the DOM to get the full list again?